databuilder/extractor/neo4j_search_data_extractor.py

```diff
@@ -25,6 +25,7 @@ class Neo4jSearchDataExtractor(Extractor):
         OPTIONAL MATCH (table)-[:COLUMN]->(cols:Column)
         OPTIONAL MATCH (cols)-[:DESCRIPTION]->(col_description:Description)
         OPTIONAL MATCH (table)-[:TAGGED_BY]->(tags:Tag)
+        OPTIONAL MATCH (table)-[:TAGGED_BY]->(badges:Tag) WHERE badges.tag_type='badge'
```

Comment:
i found if i filtered tags as "WHERE tags.tag_type='default'", the badges were no longer searchable in staging due to them not being surfaced any other way. so for now i'd like to leave tags as is

Reply:
Mmm.. not sure why that is the case as I can see that `tag_type` is either `default` or `badge`.
Is it this query not returning TAG?

```
MATCH (n:Tag)
RETURN DISTINCT(n.tag_type);
```

Reply:
so that works - the issue is that i would like to be able to search the ui on staging as either `tag:<badge_name>` or `badge:<badge_name>` while testing the feature. but if i filter out `badges` from the tags for neo4j here, then i can't find any badges. it's an issue around being in/out of sync for search service + updated definition here. i can change it now, but i was thinking i could send out a PR after i change the search service. 
testing ES directly via kibana returns what i expect fwiw. 

i'm happy to add that filter now since i do have one table i can test against in staging though.

Reply:
Got it. I am fine with both approach and please proceed as you see fit

Reply:
@csteez just saw this discussion, let's add the filter now to keep the tag/badge separate if you have one table for testing. Thanks.